src/libraries/System.Text.Encoding.CodePages/src/System/Text/SBCSCodePageEncoding.cs

```diff
@@ -32,6 +33,18 @@ public SBCSCodePageEncoding(int codePage, int dataCodePage) : base(codePage, dat
         {
         }
 
+        internal static unsafe ushort ReadUInt16(byte *pByte)
```

Comment:
Nit: standard library spacing is to use `T* param` instead of `T *param`. (Same comment on lines 40 and 44: remove the whitespace.)

Reply:
Updated and pushed